### PR TITLE
added external wallet functionality

### LINKED ISSUE #390: Port other wallet function from hydra do basilisk

### PR DESCRIPTION
**Pull request by obsivac-cleevio**
- Fixes #390 
- implementation is the same as for hydra

### CODE REVIEW

**Comment by vkulinich-cl** [comment]
Screenshot 2023-04-07 at 10 14 54
data se nenacitaji pro external walletku

**Comment by vkulinich-cl** [comment]
kdyz reloaudnu stranku, tak se odpojim z external walletky

File: src/assets/icons/PasteAddressIcon.svg
Comment (vkulinich-cl) [inline src/assets/icons/PasteAddressIcon.svg:1]: tahle ikona se nepouziva, ze?

**Comment by vkulinich-cl** [comment]
pri navigovani do Wallet tabu, query account param se zmaze z URL

**Comment by vkulinich-cl** [comment]
Screenshot 2023-04-07 at 10 25 50
Join all farms button neni deaktivovane pro external wallet

File: src/sections/wallet/connect/modal/ExternalWalletConnectModal.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                safeConvertAddressSS58(value, 0) != null ||
+                t("wallet.assets.transfer.error.validAddress"),
+            },
+          }}
+          render={({
+            field: { name, value, onChange },
+            fieldState: { error },
+          }) => {
+            return (
+              <div>
```
Comment (vkulinich-cl) [inline src/sections/wallet/connect/modal/ExternalWalletConnectModal.tsx:77]: asi zbytecny div

Comment (obsivac-cleevio) [reply src/sections/wallet/connect/modal/ExternalWalletConnectModal.tsx:77]: nn ten musi zustat

**Review by vkulinich-cl — APPROVED** [review]

**Pull request merged**